packages/cactus-plugin-ledger-connector-daml/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -0,0 +1,769 @@
+import test, { Test } from "tape-promise/tape";
```

Comment:
@raynatopedrajeta Please use jest for tests exclusively we have some old tape tests left in the codebase but we've been migrating away from tape fora while by refactoring each test case one by one. New test cases should use Jest so that we don't end up creating more work for the people working on the refactoring

Reply:
@raynatopedrajeta OK, please 1) open the ticket, 2) link to this comment from it 3) delete this test case

Reply:
Hello @petermetz ,
Here is the link to the new created ticket: https://github.com/hyperledger-cacti/cacti/issues/3707.

This file has also been deleted on my latest push. Thank you!

Rayn